db/migrate/20220429060600_create_canonical_armors_smithing_materials.rb

```diff
@@ -11,7 +11,7 @@ def change
                    null:        false,
                    foreign_key: true,
                    index:       { name: :index_canonical_armors_smithing_mats_on_canonical_mat_id }
-      t.integer    :count,
+      t.integer    :quantity,
```

Comment:
The name 'quantity' seemed more appropriate than 'count', which is more generic and less self-explanatory.